aopy/analysis.py

```diff
@@ -817,6 +817,75 @@ def calc_sem(data, axis=None):
 
     return SEM
 
+
+def calc_mean_erp(data, event_times, time_before, time_after, samplerate, subtract_baseline=True):
+    '''
+    Calculates the mean (across trials) event-related potential (ERP) for the given timeseries data.
+
+    Args:
+        data (nt, nch): timeseries data across channels
+        event_times (ntrial): list of event times
+        time_before (float): number of seconds to include before each event
+        time_after (float): number of seconds to include after each event
+        samplerate (float): sampling rate of the data
+        subtract_baseline (bool, optional): if True, subtract the mean of the aligned data during
```

Comment:
I'd suggest adding an optional input argument that allows you to specify the window of time to use for computing the baseline. Current implementation could be the default, but would increase flexibility to let it be set.

Reply:
added baseline_window

Reply:
Looks good!